### PR TITLE
Th2 4615 add urls to data metadata in commands

### PR DESCRIPTION
**Pull request by xHacka**
1. Adapters should have been changed in PR: https://github.com/th2-net/th2-ds-source-lwdp/pull/23, but since it needed a feature it's not yet in dev. After PR `handle_streams` will appear as `handle`.
3. SSEHandlerClassBase now has protected field `_urls` which will hold api request urls.
4. abstract method `_store_urls` added to SSEHandlerClassBase which will be called from parent to store the url (api request happens in child class)
    - Handle
        1. Store `urls`
        2. Use Command
5. `_sse_bytes_stream` is no longer abstract and is implemented in SSEHandlerClassBase. since after adding `_store_urls` all methods had same body.

### CODE REVIEW

**Comment by ConnectDIY** [comment]
@xHacka 
Hi
I merged https://github.com/th2-net/th2-ds-source-lwdp/pull/23
1. Can you resolve conflicts here
2. Let me don't remind you about `release notes` each time, please.

**Comment by xHacka** [comment]
Conflicts resolved.

I didnt update release notes at that commit because I wanted to get approval of implemented method.
release_notes updated (also added some previously implemented features)

**Review by ConnectDIY — CHANGES_REQUESTED** [review]

File: th2_data_services_lwdp/commands/http.py [hunk 1]
```diff
@@ -120,7 +123,13 @@ def _data_object(self, data_source: HTTPDataSource) -> Data:
              Data
         """
         sse_events_stream = partial(self._sse_events_stream, data_source)
-        return Data(sse_events_stream).map_stream(self._sse_handler.handle).use_cache(self._cache)
+        data = Data(sse_events_stream).map_stream(self._sse_handler.handle).use_cache(self._cache)
+        data.metadata["urls"] = self._urls
+        return data
+
+    @abstractmethod
+    def _store_urls(self, api) -> None:
```
[the remark this answers is not in the document]
Comment (xHacka) [reply th2_data_services_lwdp/commands/http.py:131]: Type hint HTTPAPI added

File: th2_data_services_lwdp/commands/http.py [hunk 2]
```diff
@@ -85,11 +86,13 @@ def return_data_object(self):
...
 
-    @abstractmethod
     def _sse_bytes_stream(
         self, data_source: HTTPDataSource
     ) -> Generator[bytes, None, None]:  # noqa
-        pass
+        api: HTTPAPI = data_source.source_api
+        for url in self._urls:
+            # LOG             logger.info(url)
+            yield from api.execute_sse_request(url)
```
[the remark this answers is not in the document]
Comment (xHacka) [reply th2_data_services_lwdp/commands/http.py:95]: Method changed to
`def _get_urls(self, api: HTTPAPI) -> List[str]:`

self._urls removed

**Review by ConnectDIY — APPROVED** [review]

**Pull request merged**